Source/Particles/ElementaryProcess/QEDPairGeneration.H

```diff
@@ -85,6 +85,17 @@ public:
     * lookup tables. Therefore, it should be rather lightweight to copy.
     *
     * @param[in] generate_functor functor to be called to determine the properties of the generated pairs
+    * @param[in] a_pti particle iterator to iterate over photons undergoing QED pair generation process
+    * @param[in] lev   the mesh-refinement level
+    * @param[in] ngE   ...?
```

Comment:
@EZoni or @lucafedeli88 maybe ? :)

Reply:
I'm not familiar with this module, but usually `ngE` is the number of guard cells allocated for the E and B MultiFabs. So one suggestion could be the following:
```suggestion
    * @param[in] ngE   number of guard cells allocated for the E and B MultiFabs
```
Let's see if @lucafedeli88 confirms that this is indeed the case for this specific function.